### PR TITLE
Upgrade to ES 1.7.1

### PR DESCRIPTION
**Pull request by kenoir**
This PR:
- Upgrades to ES 1.7.1
- Changes log level for aws plugin to be less noisy
- Upgrades plugins where necessary
- Remove BigDesk plugin as it is incompatible (and not used)

**TODO:**
- Update Cloudwatch logs plugin to be compatible.

### CODE REVIEW

**Comment by kenoir** [comment]
I've tested this locally and it seems to work fine with 1.7.1.

I'm unsure whether the Cloudwatch logs plugin will be compatible but trying this out in TEST will make that clear.

**Comment by theefer** [comment]
No code or config change needed at all? I thought we'd have to explicitly enable dynamic scripting or things like that?

**Comment by kenoir** [comment]
@theefer maybe about dynamic scripting. I'll double check.

File: elasticsearch/dev-install.sh [hunk 1]
```diff
@@ -27,9 +27,8 @@ then
     cd elasticsearch
     ./bin/plugin -install mobz/elasticsearch-head
     ./bin/plugin -install elasticsearch/elasticsearch-cloud-aws/2.3.0
-    ./bin/plugin -install lukas-vlcek/bigdesk
     ./bin/plugin -install karmi/elasticsearch-paramedic
-    ./bin/plugin -install com.yakaz.elasticsearch.plugins/elasticsearch-action-updatebyquery/2.2.0
+    ./bin/plugin -install com.yakaz.elasticsearch.plugins/elasticsearch-action-updatebyquery/2.6.0
```
Comment (jamesgorrie) [inline elasticsearch/dev-install.sh]: We can remove this.

File: elasticsearch/elasticsearch.yml [hunk 2]
```diff
@@ -11,7 +11,7 @@ cluster.name: media-service
 # Node names are generated dynamically on startup, so you're relieved
 # from configuring them manually. You can tie this node to a specific name:
 #
-# node.name: "Franz Kafka"
```
Comment (kenoir) [inline elasticsearch/elasticsearch.yml]: Use hostname as name (this is the private hostname containing the private ip in AWS). This makes matching instances in a cluster to their AWS identity a bit easier.

File: elasticsearch/elasticsearch.yml [hunk 3]
```diff
@@ -360,3 +360,7 @@ action.disable_delete_all_indices: true
 
 ################## ES Cloudwatch config ##################
 metrics.cloudwatch.namespace: "@@STAGE/Elasticsearch"
+
+################## Dynamic scripting ##################
```
Comment (kenoir) [inline elasticsearch/elasticsearch.yml]: Enable dynamic scripting (disabled by default in 1.7.1).

File: elasticsearch/logging.yml [hunk 4]
```diff
@@ -5,7 +5,7 @@ logger:
   action: DEBUG
 
   # aws
-  com.amazonaws: DEBUG
+  com.amazonaws: WARN
```
Comment (kenoir) [inline elasticsearch/logging.yml]: Reduce log level of noisy plugin.

File: elasticsearch/dev-install.sh [hunk 5]
```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-ELASTICSEARCH_VERSION=1.3.4
+ELASTICSEARCH_VERSION=1.7.1
```
Comment (kenoir) [inline elasticsearch/dev-install.sh]: Use latest version of ES.

File: elasticsearch/build.sh [hunk 6]
```diff
@@ -16,12 +16,10 @@ then
     tar xfv downloads/elasticsearch.tar.gz -C downloads
     mv downloads/elasticsearch-* downloads/elasticsearch
 
-    ./downloads/elasticsearch/bin/plugin -install elasticsearch/elasticsearch-cloud-aws/2.3.0
+    ./downloads/elasticsearch/bin/plugin -install elasticsearch/elasticsearch-cloud-aws/2.7.1
```
Comment (kenoir) [inline elasticsearch/build.sh]: Update plugins to work with ES 1.7.1

File: elasticsearch/elasticsearch.yml (hunk 2 above)
Comment (theefer) [inline elasticsearch/elasticsearch.yml]: Hence spoiling the only bit of fun during major incidents. What a tragedy!

Comment (kenoir) [inline elasticsearch/elasticsearch.yml]: Indeed. I will miss googling the names of marvel characters while waiting
for the cluster to stabilise. Won't miss trying to figure out which node
needs killing from the AWS console.

On Mon, 7 Sep 2015 at 13:35 Sébastien Cevey <notifications@github.com>
wrote:

> In elasticsearch/elasticsearch.yml
> <https://github.com/guardian/grid/pull/1197#discussion_r38858447>:
>
> > @@ -11,7 +11,7 @@ cluster.name: media-service
> >  # Node names are generated dynamically on startup, so you're relieved
> >  # from configuring them manually. You can tie this node to a specific name:
> >  #
> > -# node.name: "Franz Kafka"
>
> Hence spoiling the only bit of fun during major incidents. What a tragedy!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/guardian/grid/pull/1197/files#r38858447>.
>

-- 

------------------------------
Visit theguardian.com. On your mobile and tablet, download the Guardian 
iPhone and Android apps theguardian.com/guardianapp and our tablet editions 
theguardian.com/editions.  Save up to 57% by subscribing to the Guardian 
and Observer - choose the papers you want and get full digital access. 
 Visit subscribe.theguardian.com

This e-mail and all attachments are confidential and may also be 
privileged. If you are not the named recipient, please notify the sender 
and delete the e-mail and all attachments immediately. Do not disclose the 
contents to another person. You may not use the information for any 
purpose, or store, or copy, it in any way.  Guardian News & Media Limited 
is not liable for any computer viruses or other material transmitted with 
or as part of this e-mail. You should employ virus checking software.
 
Guardian News & Media Limited is a member of Guardian Media Group plc. Registered 
Office: PO Box 68164, Kings Place, 90 York Way, London, N1P 2AP.  Registered 
in England Number 908396

Comment (theefer) [inline elasticsearch/elasticsearch.yml]: :grinning:

**Comment by theefer** [comment]
w00t :+1:

**Pull request merged**